### ISSUE TITLE
query serverConfig

### ISSUE THREAD

**Issue by mmctrl** [oracle/weblogic-monitoring-exporter#83]
hi,
is this possible to configure query to serverConfig ?
I need "maxCapacity" value from
management/weblogic/latest/serverConfig/JDBCSystemResources/ds_name/JDBCResource/JDBCConnectionPoolParams

Thanks

**Comment by ddsharpe** [comment]
@russgold I don't believe that this is possible today.  Only Runtime data is configurable. Moving this to an enhancement request.  It would be beneficial to be able to graph data where the Max value is known, similar to JVM max heap, used heap.
